nemo_curator/utils/distributed_utils.py

```diff
@@ -670,7 +670,7 @@ def read_data(
                 files_per_partition=files_per_partition,
                 input_meta=input_meta,
                 columns=columns,
-                read_func_single_partition=read_func_single_partition,
+                read_func_single_partition=read_single_partition,
```

Comment:
I looked at this line and I thought this was a bug, however `read_data_files_per_partition` would've imputed it as `read_single_partition` because it was None